src/components/EvComponents/TwoColumnBox.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Grid, Box } from "@mui/material";
+import { BlockTitle } from "components/EvComponents/StyledTypography";
+import { Paragraph } from "components/EvComponents/Typography";
+import Image from "components/BazarImage";
+
+
+
+
+const TwoColumnContent = (img , title , text ,direction) => {
+    if(direction === "left"){
```

Comment:
we want the direction switching to work only on big screens.
in small screens the image should be on top and text on bottom
you might use the "order" prop in Grid component
checkout this example : https://stackoverflow.com/questions/54726871/changing-the-order-of-grid-item-stacking-in-material-ui

Reply:
Resolved, kindly have a look